peps/pep-0744.rst

```diff
@@ -133,12 +133,20 @@ Like the rest of the interpreter, the JIT compiler is generated at build time,
 and has no runtime dependencies. It supports a wide range of platforms (see the
 `Support`_ section below), and has comparatively low maintenance burden. In all,
 the current implementation is made up of about 900 lines of build-time Python
-code and 500 lines of runtime C code.
+code and 500 lines of runtime C code. The current approach is optimized for 
+runtime compilation and would likely not work well for ahead-of-time compilation. 
+This is not a direction currently being explored or pursued.
 
 Specification
 =============
 
-The JIT will become non-experimental once all of the following conditions are
+Before discussing the requirements that need to be met before the JIT can be
```

Comment:
This first sentence is a bit meta and breaks the flow of the PEP, in my opinion. You can probably just keep the second sentence and add it to the start of the following paragraph.